bin/templates/cordova/lib/builders/ProjectBuilder.js

```diff
@@ -365,6 +382,31 @@ function findOutputApksHelper (dir, build_type, arch) {
     return ret;
 }
 
+function findOutputBundlesHelper (dir, build_type) {
+    var shellSilent = shell.config.silent;
+    shell.config.silent = true;
+
+    // list directory recursively
+    var ret = shell.ls('-R', dir).map(function (file) {
```

Comment:
As said in another critic, because we will be replacing `shell` with `fs-extra` in the near future, I won't be making any changes to the usages of `shell`, but I will be replacing `var` with `const`.